crates/bevy_transform/Cargo.toml

```diff
@@ -8,10 +8,14 @@ repository = "https://github.com/bevyengine/bevy"
 license = "MIT OR Apache-2.0"
 keywords = ["bevy"]
 
+[features]
+serialize = ["serde"]
```

Comment:
This should rely on `bevy_math/serialize`, as `Affine3A` and `Vec3A` will not implement the traits without it.